packages/elements/sentry.ts

```diff
@@ -0,0 +1,13 @@
+import * as Sentry from '@sentry/browser';
+import { BrowserTracing } from '@sentry/tracing';
+import pjson from './package.json';
+
+Sentry.init({
+  dsn: 'https://1d34034cd858470ba77ff10172425af2@o1356305.ingest.sentry.io/4504792337219584',
+  environment: process.env.NODE_ENV,
```

Comment:
This shouldnt be an env var right? our clients might modify this (or not set it at all?)

Reply:
Was thinking this too, but there's ultimately no way around exposing the DSN when we publish to NPM even if we store this as env variable in the repo. That being said, as it's written clients shouldn't be able to modify it at runtime, the only way I can think would be they manually change the file in their node_modules.

TBH, the more I think about it and search the internet... I'm not sure how standard it is to include sentry in a public, published npm package. There could be some privacy and security issues.